functions/changes/index.js

```diff
@@ -8,11 +8,48 @@ const {
 } = require("osm-replication-streams");
 const osm2obj = require('osm2obj');
 const stringify = require("stringify-stream");
+const through2 = require("through2");
 
 const checkpointStream = require('./lib/checkpoint-stream');
 
 const STREAM_NAME = env.require("STREAM_NAME");
 
+/**
+ * A simplified protocol of the osm2obj format omits
+ * the action property of each OSM entity and attaches
+ * the visible property following these rules:
+ * 
+ * 1. If the action is a create, then visible=true because
+ * a created entity is a visible entity with version=1
+ * 2. If the action is a modify, then visible=true with a 
+ * version > 1
+ * 3. If the action is a delete, then visible=false
+ * 
+ * @param {Object} obj - An osm2obj object
+ * @returns {Object} A simplified object
+ * 
+ */
+function obj2simple (obj) {
+  let res = {};
+
+  // Omit the action
+  for (var prop in obj) {
+    if (prop !== 'action') {
+      res[prop] = obj[prop];
+    }
+  }
+
+  if (obj['action']) {
+    if (obj['action'] === 'create' || obj['action'] === 'modify') {
+      res['visible'] = 'true';
+    }
+    else {
+      res['visible'] = 'false';
+    }
+  }
+  return res;
```

Comment:
```javascript
// destructure to avoid mutation
const { action, ...res } = {
  ...obj,
  visible: ["create", "modify"].includes(obj.action)
};

return res;
```

instead?

(Better to return primitive booleans for `visible` if nothing else.)

Reply:
@mojodna sure! I was just following [the wiki](http://wiki.openstreetmap.org/wiki/Elements) and thought visible had to be a string for possible interoperability. What do you think?

Reply:
^ this looks simple and succinct, I wonder though if we should guard against `obj.action` not existing?

Reply:
We get to make up the format ;-)

`obj.action` should always exist, though maybe a more general TODO should be to hook it up to Sentry in case there are other unexpected inputs.